src/extra/imgui.cpp

```diff
@@ -3755,10 +3754,10 @@ struct Disable : public Base {
   void setParam(int index, const CBVar &value) {
     switch (index) {
     case 0:
-      _disable = value;
+      _blocks = value;
       break;
```

Comment:
Breaking change, no impact but we should take note somehow of those (even a trello board or whatever would do)

Reply:
Should we start to tag some version?

Reply:
Yes we should and prob create a changelog.md for those changes